recipes/seqan3/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 requirements:
   build:
     - make
-    - cmake >=3.14
+    - cmake >=3.7
```

Comment:
Hu. Do we need a rebuild for cmake?

Reply:
I don't think so, I'm not sure how this got triggered.

Reply:
Probably just some "concurrency issue" ;).
From the commit/merge and autobump CI dates, it looks to me that the bot's 3.0.3 update branch had been superseded by a manual PR which updated it to `cmake >=3.14`. The next autobump run then started before that PR was merged and finished after it was merged and thus effectively wants to revert the `>=3.14` because it started with `>=3.7`, I'd guess.